cmds/core/sluinit/uinit.go

```diff
@@ -110,3 +113,68 @@ func unmountAndExit() {
 	time.Sleep(5 * time.Second) // let queued up debug statements get printed
 	os.Exit(1)
 }
+
+// scanIscsiDrives calls iscsinl to mount iscsi drives.
+// format: netroot=iscsi:@X.Y.Z.W::3260::iqn.FOO.com.abc:hostname-boot
+func scanIscsiDrives() error {
+
+	slaunch.Debug("Scanning kernel cmd line for *netroot* flag")
+	val, ok := cmdline.Flag("netroot")
+	if !ok {
+		return errors.New("netroot flag is not set")
+	}
+
+	// val = iscsi:@10.196.210.62::3260::iqn.1986-03.com.sun:ovs112-boot
+	slaunch.Debug("netroot flag is set with val=%s", val)
+	s := strings.Split(val, "::")
```

Comment:
there's stuff in pkg/dhclient to parse this format. you could export that and use it here?

Reply:
perhaps it's something we should move to iscsinl?